docs/RFCS/online_schema_change.md

```diff
@@ -131,8 +131,19 @@ Since schema change operations are potentially long running they need
 to be restartable or abortable if the node performing them dies. We
 accomplish this by performing the schema change operation for a table
 on a well known node: the replica holding the leader lease for the
-first range of the table (i.e. containing the key `/<tableID>`). When
-a node receives a schema change operation such as `CREATE INDEX` it
+first range of the table (i.e. containing the key `/<tableID>`), known
+as the "table leader". Each node discovers via gossip the list of
+all configured tables, and starts a table leader goroutine
+for each table it is a leader for. A table leader goroutine
+executes schema changes on its table, and exits
+as soon as it discovers it has lost its leader lease.
+
+It is critical, that the goroutine chunk its processing into small
+enough chunks, to allow it to relinquish leadership in a timely manner.
+While it is safe to apply the same schema change concurrently, it is
+unsafe to have two different schema changes be processed concurrently.
```

Comment:
I would prefer to just go foolproof for now. Since we're going to be doing this backfill transactionally (at least for a while), designating a per-table key into which the worker `CPut`s the the lease with each transactional unit of work performed is enough (if the `CPut` comes back negative, someone else has taken over). This goes nicely with keeping state about performed work, which is listed as "unresolved" below but seems like a useful and straightforward addition.

Later, if we manage to get rid of the transaction or for some other reason don't want to use that key as a coercion point, we can send the leader lease along with the backfill requests as a stipulation that they be executed only under that lease.